rs/nns/governance/canister/governance.did

```diff
@@ -491,10 +505,21 @@ type ManageNeuronRequest = record {
   id : opt NeuronId;
   command : opt ManageNeuronCommandRequest;
   neuron_id_or_subaccount : opt NeuronIdOrSubaccount;
+
+  // Controls how the resulting_neuron field in the response is populated.
+  // Possible values here and how the response will be populated:
+  //
+  //     * null - Not populated.
+  //     * 1 - NeuronInfo
+  //     * 2 - NueronInfo, but "big" fields are cleared. To wit,
+  //         * recent_ballots
+  //         * known_neuron_data
+  resulting_neuron_info : opt int32;
```

Comment:
is the idea here that we can return the info about the neuron in some replies and this is why we add it?

Reply:
Yes. I thought you said we want this so that UIs can warn users? I would rather not have resulting_neuron_info, because the API is already sufficient (without this) for UIs to detect that the user has neurons need to be refreshed soon.

Reply:
Note that this only helps if the neuron being touched is in need of refresh. If the user has other neurons in that situation, the caller would not find that out from the response. Not sure if this is sufficient for the use cases we care about...

Reply:
See also the [other thread about resulting_neuron_info][other-thread].

[other-thread]: https://github.com/dfinity/ic/pull/2525#discussion_r1835985178